api/actions/app_logs_test.go

```diff
@@ -209,7 +209,7 @@ var _ = Describe("ReadAppLogs", func() {
 		})
 	})
 
-	When("GetLatestBuildByAppGUIDReturns returns a Forbidden error", func() {
+	When("GetLatestBuildByAppGUID returns a Forbidden error", func() {
```

Comment:
`GetLatestBuildByAppGUIDReturns` is the name of the function that is returning the error.

Reply:
The function name is GetLatestBuildByAppGUID and the mock function to set the return value is GetLatestBuildByAppGUIDReturns right? Which should we use in the When statement?